x/concentrated-liquidity/incentives_test.go

```diff
@@ -870,3 +874,655 @@ func (s *KeeperTestSuite) TestGetInitialUptimeGrowthOutsidesForTick() {
+			poolSetup:                    true,
+			lowerTick:                    0,
+			upperTick:                    2,
+			currentTick:                  1,
+			lowerTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			upperTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			globalUptimeGrowth:           uptimeHelper.threeHundredTokensMultiDenom,
+
+			// Since current tick is within range, we expect global - upper - lower
+			expectedUptimeGrowthInside: uptimeHelper.hundredTokensMultiDenom,
+			expectedError:              false,
+		},
+		"upper tick > current tick > lower tick, nonzero uptime growth inside (wider range)": {
+			poolSetup:                    true,
+			lowerTick:                    -19753,
+			upperTick:                    8921,
+			currentTick:                  -97,
+			lowerTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			upperTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			globalUptimeGrowth:           uptimeHelper.threeHundredTokensMultiDenom,
+
+			// Since current tick is within range, we expect global - upper - lower
+			expectedUptimeGrowthInside: uptimeHelper.hundredTokensMultiDenom,
+			expectedError:              false,
+		},
+		"upper tick > current tick > lower tick, zero uptime growth inside (nonempty trackers)": {
+			poolSetup:                    true,
+			lowerTick:                    0,
+			upperTick:                    2,
+			currentTick:                  1,
+			lowerTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			upperTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
+			globalUptimeGrowth:           uptimeHelper.twoHundredTokensMultiDenom,
+
+			// Since current tick is within range, we expect global - upper - lower
+			expectedUptimeGrowthInside: uptimeHelper.emptyExpectedAccumValues,
+			expectedError:              false,
+		},
+		"upper tick > current tick > lower tick, zero uptime growth inside (empty trackers)": {
+			poolSetup:                    true,
+			lowerTick:                    0,
+			upperTick:                    2,
+			currentTick:                  1,
+			lowerTickUptimeGrowthOutside: uptimeHelper.emptyExpectedAccumValues,
+			upperTickUptimeGrowthOutside: uptimeHelper.emptyExpectedAccumValues,
+			globalUptimeGrowth:           uptimeHelper.emptyExpectedAccumValues,
+
+			// Since current tick is within range, we expect global - upper - lower
+			expectedUptimeGrowthInside: uptimeHelper.emptyExpectedAccumValues,
+			expectedError:              false,
+		},
+
+		// current tick below range
+
+		"current tick < lower tick, nonzero uptime growth inside": {
+			poolSetup:                    true,
+			lowerTick:                    0,
+			upperTick:                    1,
+			currentTick:                  -1,
+			lowerTickUptimeGrowthOutside: uptimeHelper.twoHundredTokensMultiDenom,
+			upperTickUptimeGrowthOutside: uptimeHelper.hundredTokensMultiDenom,
```

Comment:
Since if curTick is below the range, both the lower and upper ticks are "facing away" from the current tick (meaning that `UptimeGrowthOutside` represents uptime growth _above_ the tick). Thus, lowerTick's `UptimeGrowthOutside` must include upperTick's `UptimeGrowthOutside` and therefore we can assert in this case that:

`lowerTick.UptimeGrowthOutside >= upperTick.UptimeGrowthOutside`